charts/plex-media-server/values.yaml

```
@@ -44,6 +44,12 @@ pms:
   # NOTE: When set, 'configStorage' and 'storageClassName' are ignored.
   configExistingClaim: ""
 
+  # Providing both values will add in the PLEX_CLAIM environment variable
+  # with the correct secretKeyRef configuration
+  # Clashes with providing PLEX_CLAIM via `extraEnv` so only provide one or the other!
+  claimSecretName: ""
+  claimSecretValue: ""
```

nit: we could maybe use a nested key struct for these, eg:

```suggestion
  claimSecret:
    name: ""
    key: ""
```